packages/expo-file-system/src/next/pathUtilities/url.ts

```diff
@@ -70,7 +70,7 @@ function getPathFromURLPosix(url) {
   const pathname = url.pathname;
   for (let n = 0; n < pathname.length; n++) {
     if (pathname[n] === '%') {
-      const third = pathname.charAt(pathname, n + 2) | 0x20;
+      const third = pathname.charAt(n + 2) as any | 0x20;
```

Comment:
```suggestion
      const third = +pathname.charAt(n + 2) | 0x20;
```
it should produce the same result, but as least we don't need to cast to `any`